src/modules/UI.js

```diff
@@ -0,0 +1,43 @@
+import TrashIcon from '../assets/icons/trash-alt-regular.svg';
+import SyncIcon from '../assets/icons/sync-alt-solid.svg';
+import ArrowIcon from '../assets/icons/arrow-right-solid.svg';
+import EllipsisIcon from '../assets/icons/ellipsis-v-solid.svg';
+import CheckBOXHandler from './CheckBoxHandler.js';
+
+export default class UI {
+  constructor(store) {
+    this.checkBOXHandler = new CheckBOXHandler(store);
+    this.store = store;
+  }
+
+  updateIcons() {
+    this.syncIcon = document.querySelector('.sync-icon');
+    this.syncIcon.src = SyncIcon;
+
+    this.arrowIcon = document.querySelector('.arrow-icon');
+    this.arrowIcon = ArrowIcon;
+  }
+
+  generateHtmlTasks() {
+    this.todoListContainer = document.getElementById('todo-list-container');
+    this.todoListContainer.innerHTML = '';
+
+    this.updateIcons();
+
+    this.store.getItems().forEach((obj, index) => {
+      const checked = (obj.completed) ? 'checked' : '';
+      const lineTrought = (obj.completed) ? 'line-trought' : '';
+      obj.index = index + 1;
+
+      this.todoListContainer.innerHTML += `<div id="${obj.index}"class="d-flex justify-content-start align-items-center mt-1">
+      <div class="form-check me-1">
+        <input class="checkbox form-check-input" type="checkbox" id="blankCheckbox" value="true" aria-label="..." ${checked}>
+      </div>
+      <input id="text-input-${obj.index - 1}" class="text-description ${lineTrought}" type="text" placeholder="${obj.description}" value="${obj.description}"></input>
+      <img src="${EllipsisIcon}" class="icon ellipsis-icon fas fa-ellipsis-v ms-auto text-secondary"/>
+      <img src="${TrashIcon}" class="icon trash-icon ms-auto text-secundary d-none"/>
+    </div>`;
+    });
+    this.checkBOXHandler.addChangeListner();
+  }
+}
```

Comment:
Your UI module is not containing a single delete and edit button, your app will look more professional if those functionalities are added 🤨.